templates/taxi/car_detail.html

```diff
@@ -12,7 +12,14 @@ <h1>
     </a>
   </h1>
   <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
-  <h1>Drivers</h1>
+  <h1>Drivers 
+    {% if user not in car.drivers.all %}
+    <a style="float: right" class="btn btn-primary" href="{% url 'taxi:driver-assign-to-car' user.id car.id %}">
```

Comment:
The `url` tag is missing a comma between `user.id` and `car.id`. It should be `{% url 'taxi:driver-assign-to-car' user.id car.id %}`.